hermes-console-vue/src/views/admin/consumer-groups/consumer-group-member/ConsumerGroupMembers.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+  import { formatNumber } from '@/utils/number-formatter/number-formatter';
+  import { useI18n } from 'vue-i18n';
+  import type { ConsumerGroupMember } from '@/api/consumer-group';
+  const { t } = useI18n();
+
+  const props = defineProps<{
+    members: ConsumerGroupMember[];
+  }>();
+</script>
+
+<template>
+  <v-table>
+    <thead>
+      <tr>
+        <th class="text-left">{{ t('consumerGroups.listing.type') }}</th>
+        <th class="text-left">
+          {{ t('consumerGroups.listing.partition') }}
+        </th>
+        <th class="text-left">
+          {{ t('consumerGroups.listing.currentOffset') }}
+        </th>
+        <th class="text-left">
+          {{ t('consumerGroups.listing.endOffset') }}
+        </th>
+        <th class="text-left">{{ t('consumerGroups.listing.lag') }}</th>
```

Comment:
```suggestion
        <th class="text-left">
          {{ t('consumerGroups.listing.type') }}
        </th>
        <th class="text-left">
          {{ t('consumerGroups.listing.partition') }}
        </th>
        <th class="text-left">
          {{ t('consumerGroups.listing.currentOffset') }}
        </th>
        <th class="text-left">
          {{ t('consumerGroups.listing.endOffset') }}
        </th>
        <th class="text-left">
          {{ t('consumerGroups.listing.lag') }}
        </th>
```
minor maybe like that more consistently